### DISCUSSION TITLE
How to add loras in termux

### DISCUSSION THREAD

**Discussion by KintCark** [rupeshs/fastsdcpu#166] [General]
How do I add loras in termux fastsd what's the functions like what do I type?

**Comment by rupeshs** [comment]
@KintCark Download lora weight and move it to `lora_models ` folder and  restart webui.
Please refer this https://github.com/rupeshs/fastsdcpu?tab=readme-ov-file#how-to-use-lora-models

**Reply by KintCark** [reply]
> @KintCark Download lora weight and move it to `lora_models ` folder and  restart webui.
> Please refer this https://github.com/rupeshs/fastsdcpu?tab=readme-ov-file#how-to-use-lora-models
> 

Hey OK I figured it out I can navigate my phone to termux folder and copy it there but I have another question is fastsd offline ? Cuz when I tick the offline box in general settings and try to create an image it say error. Also can I load different checkpoints? If I can? How do I load different checkpoints?
